packages/loopring_v3/contracts/impl/libexchange/ExchangeTokens.sol

```diff
@@ -68,7 +68,7 @@ library ExchangeTokens
         address tokenAddress,
         uint    amountToBurn
         )
-        public
+        internal
```

Comment:
Any reason to make this internal? This inlines the function in the places it is being used, to it will add extra bloat there, increasing contract sizes. Gas use for this function call isn't a concern, so best to only use `internal` for critical functions where it is.

Reply:
Reverted